plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -15,9 +15,32 @@ const languageDisplayNames = {
 	scss: 'SCSS',
 	sass: 'Sass',
 	md: 'Markdown',
-	markdown: 'Markdown'
+	markdown: 'Markdown',
+	'atomics-filter': 'Atomics',
+	atomics: 'Atomics'
 };
 
+let filterIds = 0;
+
+function createFilterableCodeBlock(code, language, displayName) {
+	filterIds++;
+	return `
+	<div class="code-block margin-top-xs" style="min-height: 30vh">
+		
+		<div class="code-block-header">
+			<span class="code-block-header-language" data-hljs-language="${language}">${displayName}</span>
+			<input
+			class="code-block-header-filter align-self-center"
+			data-code-filter-input="${filterIds}"
+			placeholder="Filter ..."
+			type="search" />
+		</div>
+		<div class="code-block-body" style="max-height: 20vh; overflow-y: scroll;">
```

Comment:
I think we have to change max-height as 30vh in order to achieve the desired result !

Reply:
Updated!